src/adv.py

```diff
@@ -37,14 +39,43 @@
 # Main
 #
 
-# Make a new player object that is currently in the 'outside' room.
 
+
+#Instantiate a few items
+sword = Item('sword', "this is a magical sword")
+phone = Item('phone', 'this is the best phone')
+coins = Item('coins', 'these coins can buy you more items')
+
+# Make a new player object that is currently in the 'outside' room.
+alejandrok = Player('Alejandro', room['outside'])
+alejandrok.currentRoom.items = [sword, phone, coins]
 # Write a loop that:
 #
 # * Prints the current room name
 # * Prints the current description (the textwrap module might be useful here).
 # * Waits for user input and decides what to do.
-#
+
+while True:
```

Comment:
Great use of the while loop, just make sure to watch how you use it, as it can increase the time and space complexity of the file. Overall great job!